crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/AssessmentSummaryService.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.justice.laa.crime.meansassessment.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.meansassessment.dto.maatcourtdata.HardshipReviewDTO;
+import uk.gov.justice.laa.crime.meansassessment.dto.maatcourtdata.IOJAppealDTO;
+import uk.gov.justice.laa.crime.meansassessment.dto.maatcourtdata.PassportAssessmentDTO;
+import uk.gov.justice.laa.crime.meansassessment.model.common.ApiAssessmentSummary;
+import uk.gov.justice.laa.crime.meansassessment.model.common.ApiCreateMeansAssessmentResponse;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.AssessmentType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CurrentStatus;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.ReviewType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.WorkType;
+
+import java.util.ArrayList;
+
+import static java.util.Optional.ofNullable;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AssessmentSummaryService {
+
+    private final MaatCourtDataService maatCourtDataService;
+
+    public ApiCreateMeansAssessmentResponse addAssessmentSummaryToMeansResponse(final ApiCreateMeansAssessmentResponse assessmentResponse,
+                                                                                final String laaTransactionId) {
+        log.info("Generating assessment summary for means assessment response");
+        try {
+            var assesmentsSummary = new ArrayList<ApiAssessmentSummary>();
+            assesmentsSummary.add(getFinancialAssessmentSummary(assessmentResponse));
```

Comment:
`assesmentsSummary` should be `assessmentsSummary`

Reply:
yes typo. done